libs/widget-react/README.md

```diff
@@ -30,12 +29,9 @@ const cowSwapWidgetParams: CowSwapWidgetParams = {
 }
 
 const cowSwapWidgetSettings: CowSwapWidgetSettings = {
-  urlParams: {
-    chainId: 1,
-    tradeType: 'swap',
-    env: 'local',
-  },
-  appParams: {},
+  chainId: 1,
+  tradeType: 'swap',
+  env: 'local',
```

Comment:
maybe i wouldn't add the env in the basic example. Maybe not even the trade type. The simpler the better.

Reply:
do we still make the distinction params/setting ?

Reply:
Yes, I only merged `appParams` and `urlParams` into `CowSwapWidgetSettings`. But I would keep distinction between `CowSwapWidgetParams` and `CowSwapWidgetSettings`.
`CowSwapWidgetParams` contains static data like a `container`, `metaData` and `provider`, whereas `CowSwapWidgetSettings` is full of parameters that might me changed on flight.

Reply:
>maybe i wouldn't add the env in the basic example

Makes sense. Made `CowSwapWidgetSettings` fully optional